pkg/crc/machine/start.go

```diff
@@ -112,12 +112,57 @@ func (client *client) updateVMConfig(startConfig types.StartConfig, vm *virtualM
 	return nil
 }
 
-func growRootFileSystem(sshRunner *crcssh.Runner, preset crcPreset.Preset, persistentVolumeSize int) error {
-	rootPart, err := getrootPartition(sshRunner, preset)
+func restartHost(ctx context.Context, vm *virtualMachine, sshRunner *crcssh.Runner) error {
+	if err := vm.Driver.Stop(); err != nil {
+		return errors.Wrap(err, "Error re-starting machine at stop")
+	}
+	if err := startHost(ctx, vm); err != nil {
+		return errors.Wrap(err, "Error re-starting machine at start")
+	}
+	logging.Debug("Waiting until ssh is available")
+	if err := sshRunner.WaitForConnectivity(ctx, 300*time.Second); err != nil {
+		return errors.Wrap(err, "Failed to connect to the CRC VM with SSH -- virtual machine might be unreachable")
+	}
+	return nil
+}
```

Comment:
This restart means add more time to startup even it takes ~1 mins.

Reply:
yes, its less than 1 min, but that's on linux, on windows i expect it to be even longer than 1 minute

Reply:
running `partprobe` was not enough, it needed the reboot, yeah i agree that this makes it complicated and now it works because the VM ip doesn't change after a reboot